src/ensembl/src/content/app/entity-viewer/gene-view/components/proteins-list/proteins-list-item-info/ProteinsListItemInfo.tsx

```diff
@@ -66,18 +68,25 @@ const ProteinsListItemInfo = (props: Props) => {
     transcriptWithProteinDomains,
     setTranscriptWithProteinDomains
   ] = useState<Transcript | null>(null);
-  const [proteinSummary, setProteinSummary] = useState<
-    ProteinSummary | null | undefined
+
+  const [xref, setXref] = useState<Xref | null | undefined>();
```

Comment:
the `null | undefined` looks suspicious — you'll probably ever need only one of those.